modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -46,13 +54,86 @@ protected String addNewImports(String newContent) {
 		return newContent;
 	}
 
+	protected String addNewImportsJSP(String newContent, String... newImports) {
```

Comment:
@TamyrisTorres, do we really need to create this new method or can we refactor and reuse the method  `addNewImports`?

Reply:
resolved